src/Symfony/Component/Serializer/Normalizer/UnwrappingDenormalizer.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Symfony\Component\Serializer\Exception\NotNormalizableValueException;
+use Symfony\Component\Serializer\SerializerAwareInterface;
+use Symfony\Component\Serializer\SerializerInterface;
+
+class UnwrappingDenormalizer implements DenormalizerInterface, SerializerAwareInterface
+{
+    const UNWRAP_PATH = 'unwrap_path';
+
+    /**
+     * @var SerializerInterface|DenormalizerInterface
+     */
+    private $serializer;
+
+    /**
+     * @var PropertyAccess
+     */
+    private $propertyAccess;
+
+    public function __construct()
+    {
+        $this->propertyAccess = PropertyAccess::createPropertyAccessor();
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @throws NotNormalizableValueException
+     */
+    public function denormalize($data, $class, $format = null, array $context = [])
+    {
+        $propertyPath = $context[self::UNWRAP_PATH];
+        unset($context[self::UNWRAP_PATH]);
```

Comment:
Why? It could be an useful information for other (custom) denormalizers.

Reply:
This was done according to suggestion of @ogizanagi here https://github.com/symfony/symfony/pull/28887#issuecomment-438760973

Should i add some `"unwrapped"` key to `$context` to avoid circular reference instead?